### ISSUE TITLE
Broken compilation of super class calls

### ISSUE THREAD

**Issue by OmerMor** [icefallgames/SCICompanion#6]
(reported by NewRisingSun in http://sciprogramming.com/community/index.php?topic=1420.msg10242#msg10242)

When recompiling the decompiled script 998 (original name "ACTOR.SC") of SQ1, the following sequence in `Actor::setLoop`:

	(method (setLoop param1 &tmp theLooper)
		(if
			(= theLooper
				(cond 
					((== argc 0) (super setLoop:) 0)
					((not (IsObject param1)) (super setLoop: param1 &rest) 0)
					((& (param1 -info-?) $8000) (param1 new:))
					(else param1)
				)
			)
			(if looper (looper dispose:))
			((= looper theLooper) init: self &rest)

is properly decompiled, but after recompilation, it becomes:

	(method (setLoop param1 &tmp theLooper)
		(if
			(= theLooper
				(cond 
					((== argc 0) 0)
					((not (IsObject param1)) 0)
					((& (param1 -info-?) $8000) (param1 new:))
					(else param1)
				)
			)
			(if looper (looper dispose:))
			((= looper theLooper) init: self &rest)
		)

In other words,  the "`super setLoop`" in the first two conds somehow gets lost, which results in the wrong loops being displayed in SQ1's intro. 

Right now, I am working around this bug by rewriting the code as

	(method (setLoop param1 &tmp theLooper)
		(cond	((== argc 0) (super setLoop:) (= theLooper 0))
				((not (IsObject param1)) (super setLoop: param1 &rest) (= theLooper 0))
				((& (param1 -info-?) $8000) (= theLooper (param1 new:)))
				(else (= theLooper param1))
		)	
		(if theLooper
			(if looper (looper dispose:))
			((= looper theLooper) init: self &rest)
		)
	)
